EPRN.PRNS.API/Services/PrnService.cs

```diff
@@ -74,9 +74,10 @@ public async Task<StatusAndProducerDto> GetStatusWithProducerName(int id)
 
         public async Task SaveCheckYourAnswers(int id)
         {
+            string reason = "Update";
             await _prnRepository.UpdatePrnStatus(
                 id, 
-                PrnStatus.CheckYourAnswersComplete);
+                PrnStatus.CheckYourAnswersComplete, reason);
```

Comment:
I think we should remove the Required attribute from the history reason field. If we are adding history where we do not need a reason, there's no point in creating random text to satisfy the not null behaviour. It just means it should be a nullable field.

Reply:
Done.